src/pyff/schema/saml-metadata-trustinfo-v1.0.xsd

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schema  
+ targetNamespace="https://seamlessaccess.org/NS/trustinfo"
+ xmlns="http://www.w3.org/2001/XMLSchema"
+ xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata"
+ xmlns:ti="https://seamlessaccess.org/NS/trustinfo"
+ elementFormDefault="unqualified"
+ attributeFormDefault="unqualified"
+ blockDefault="substitution"
+ version="1.0">
+
+ <annotation>
+   <documentation>
+     Document title: Metadata Extension Schema for SAML V2.0 Metadata Extensions for Trust Information 1.0
+     Document identifier: metadata-trustinfo-v1.0.xsd
+     Location: http://docs.oasis-open.org/security/saml/Post2.0/
```

Comment:
What is the Location URL for? If it's where the definitive version of this schema will live, then I can't imagine that the OASIS site is the correct Location. Because the namespace is now https://seamlessaccess.org/NS/trustinfo, it looks like somewhere on the Seamless Access site would be appropriate.

Reply:
We should really put this on REFEDS namespace...

Reply:
Lets change this to seamlessaccess.org/NS/trustinfo for now @enriquepablo